pkg/labels/matcher.go

```diff
@@ -110,3 +110,8 @@ func (m *Matcher) Inverse() (*Matcher, error) {
 	}
 	panic("labels.Matcher.Matches: invalid match type")
 }
+
+// GetRegex returns the regex string.
+func (m *Matcher) GetRegex() string {
```

Comment:
We should check for `m.re` being `nil` and return `""` in that case to avoid any panics.

PS: `GetRegexString` would be better IMO.

Reply:
Done